src/waveform/renderers/waveformsignalcolors.h

```diff
@@ -45,6 +48,7 @@ class WaveformSignalColors {
     QColor m_playedOverlayColor;
     QColor m_passthroughOverlayColor;
     QColor m_bgColor;
+    int m_dimBrightThreshold = 127;
```

Comment:
We avoid initialize value in the class definition, because this introduces a hidden static constructor.  
Please move this to the constructor and use kDefaulDimBrightThreshold.